src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -146,6 +182,15 @@ function delete_local_branch_needs_force {
 }
 
 
+# Returns the previously checked out branch name
+function previous_branch_name {
+  git rev-parse --abbrev-ref "@{-1}" > /dev/null 2>&1
+  if (($? == 0)); then
+    git rev-parse --abbrev-ref "@{-1}"
```

Comment:
If there is no previous branch (or if it can't be resolved by git), you'll get an error like this:

```bash
$ git rev-parse --abbrev-ref "@{-1}"
@{-1}
fatal: ambiguous argument '@{-1}': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Since we can't really use this, we need to suppress the error and only return the branch if the command succeeded